draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3964,19 +3970,25 @@ Upgrade: websocket
        validator stored in its cache entry for the representation, and</li>
    <li>That cache entry includes a <x:ref>Date</x:ref> value, which gives the
        time when the origin server sent the original response, and</li>
-   <li>The presented Last-Modified time is at least 60 seconds before
-       the Date value.</li>
+   <li><ul>
+      <li>The presented Last-Modified time is at least one second before the Date
+      value, and the cache has reason to believe that they were generated by the
+      same clock, or</li>
+      <li>there is enough difference between the presented
+      Last-Modified time and Date value to make clock synchronisation issues
+      unlikely, in the judgement of the cache.</li>
+   </ul></li>
 </ul>
 <t>
    This method relies on the fact that if two different responses were
    sent by the origin server during the same second, but both had the
    same Last-Modified time, then at least one of those responses would
-   have a <x:ref>Date</x:ref> value equal to its Last-Modified time. The
-   arbitrary 60-second limit guards against the possibility that the Date and
+   have a <x:ref>Date</x:ref> value equal to its Last-Modified time. Previous
+   revisions of this specification imposed an
+   arbitrary 60-second limit to guard against the possibility that the Date and
    Last-Modified values are generated from different clocks or at somewhat
-   different times during the preparation of the response. An
-   implementation &MAY; use a value larger than 60 seconds, if it is
-   believed that 60 seconds is too short.
+   different times during the preparation of the response. This specification
+   has relaxed that to allow reasonable discretion.
```

Comment:
I would move the last two sentences to the changes from 7232 section to make that section a little more clear, rather than leave them here. It isn't going to help new implementations.

Reply:
Also, check the build failing. It might be due to nested list needing different tags in rfc xml.

Reply:
IIRC, the inner <ul> has to be outside the <li>, maybe.